runtime/lua/vim/treesitter.lua

```diff
@@ -240,15 +236,21 @@ function M.get_captures_at_position(bufnr, row, col)
       if M.is_in_node_range(node, row, col) then
         local c = q._query.captures[capture] -- name of the capture in the query
         if c ~= nil then
-          table.insert(matches, { capture = c, priority = metadata.priority })
+          table.insert(
+            matches,
+            {
+              capture = c,
+              priority = metadata.priority or ('' .. vim.highlight.priorities.treesitter),
```

Comment:
Is this just to coerce the string?

If so then wouldn't `tostring(...)` be better?

Reply:
Forgot about that. Yes. But even better would be for `priority` _not_ to be a string in the first place.